tests/conftest.py

```diff
@@ -83,6 +85,7 @@ def post_request_callback(request, context):
         "requestContext": {"http": {"method": request.method}},
         "body": request.body.decode(),
     }
+    reload(lambdas.webhook)
```

Comment:
Good catch on this, I need to do something similar with one of the `wiley-deposits` fixtures